public/js/views/window-view.js

```diff
@@ -24,7 +24,7 @@ var WindowView = Backbone.View.extend({
 
     evt.preventDefault();
 
-    require( '../router' ).goTo( targetUrl );
+    require( '../router' ).nav( targetUrl );
```

Comment:
Verdict: no. This feature is designed to hijack only links starting with `#`, i.e. those that use Stateman's own hash: In this app I prefer to have the hrefs be valid server routes, because we still anticipate letting the server render out this data in some scenarios.